ADO/ADOLibrary/Repositories/OrderRepository.cs

```diff
@@ -0,0 +1,184 @@
+﻿using ADOLibrary.Interfaces;
+using ADOLibrary.Models;
+using System.Data.SqlClient;
+using System.Data;
+using Z.BulkOperations;
+
+namespace ADOLibrary.Repositories
+{
+    public class OrderRepository : IOrderRepository
+    {
+        private string ConnectionString { get; set; }
+        public OrderRepository(DbConfig dbConfig)
```

Comment:
Please use something like IConnectionOptions interface which include connection string there. By that way you can create any connection options here.